input/fsh/Clinical datapoints profiles/Extensions.fsh

```diff
@@ -0,0 +1,20 @@
+Extension: EducationLevel
+Id: EducationLevel
+Title:  "Education Level Extension"
+Description: "An extension for representing the level of education of a patient."
```

Comment:
See https://confluence.hl7.org/display/CIC/%28ICHOM%29+Breast+Cancer+PCOM+Set+FHIR+IG%2C+June+22nd+2022, we need to mention that these extensions are aligned with US Core